modules/ROOT/pages/vpn-risk.adoc

```diff
@@ -0,0 +1,52 @@
+= Anypoint VPN risk
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-disabled.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+Every VPN provisioned in Mulesoft has a risk associated depending on the state of the tunnels of the VPN.
+
+For VPNs that are in High risk, MuleSoft recommends that customers plan to deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with AWS delivered auto-updates. Please refer the xref:vpn-maintenance.adoc[VPN Maintenance] for more details.
```

Comment:
```suggestion
For High risk VPNs, deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with AWS-delivered auto-updates. Refer to xref:vpn-maintenance.adoc[VPN Maintenance] for more details.
```